src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,150 @@
+package duke.main;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class Duke {
+    static ArrayList<Task> taskList = new ArrayList<Task>();
```

Comment:
I think you're missing an access modifier here. Should it be `public` or `private` etc?